CONTRIBUTING.md

```diff
@@ -0,0 +1,74 @@
+# Contributing to Nextstrain
+
+Nextstrain is an open-source and open-science project and we welcome community contributions. This includes contributions of bug reports, feature requests, data submission, scientific interpretation, language translations, and, of course, software development work across our repositories.
+
+We particularly encourage contributors from underrepresented backgrounds to engage and join our community. Please note that all Nextstrain projects follow and enforce the [Contributor Covenant Code of Conduct](https://www.contributor-covenant.org/).
+
+# Contributing code
```

Comment:
Are you ok with me demoting these to second-level headings, etc. for all the levels that follow? We should probably keep the standard of using h1 once for a title.